### PR TITLE
doc: container-image-exporter plugin doc

### PR DESCRIPTION
**Pull request by bhushan-nemade-dt**
docs for #8

### CODE REVIEW

File: docs/user-guide/plugins/container-image-exporter.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+## Steps
+1. Go to **Applications** → **Devtron Apps**.
+2. Click your application.
+3. Go to **App Configuration** → **Workflow Editor**.
+4. Click **New Workflow** and navigate to the **Build and Deploy from Source Code**.
+5. Fill the required fields in the **Create build pipeline** window and navigate to the **Post-Build stage**.
+
+{% hint style="warning" %}
+If you have already configured workflow, edit the deployment pipeline, and navigate to **Post-Build stage**.
```
Comment (abhibhaw) [inline docs/user-guide/plugins/container-image-exporter.md]: Edit the `deployment` pipeline and navigate to post `build` how?

File: docs/user-guide/plugins/container-image-exporter.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+### Input Variables
+
+| Variable                 | Format       | Description | Sample Value |
+| ------------------------ | ------------ | ----------- | ------------ |
+|   Platform               | STRING       | Specify the CPU architecture for the exported images (arm64 or amd64)            |   amd64           |
+|  AwsRegion               | STRING       | Enter the AWS region where your S3 bucket is located                             |     ap-south-1         |
+|   FilePrefix             | STRING       | Add an prefix to your exported image file name  |  temp_            |
+|   AwsAccessKey           | STRING       | Provide your AWS Access Key ID for authentication         |  VtbXliYXNlNjR2YWx1            |
+|   AwsSecretKey           | STRING       | Enter your AWS Secret Access Key for authentication        | VtudqygefuyqgjR29283bcq            |
+|   AzureAccountKey        | STRING       | Provide the Account Key for your Azure Storage account     |   abc123def567ghi           |
```
Comment (abhibhaw) [inline docs/user-guide/plugins/container-image-exporter.md]: Ig one of the AWS, GCP or Azure keys are required, this should be specified

**Review by abhibhaw — CHANGES_REQUESTED** [review]

File: docs/user-guide/plugins/container-image-exporter.md (hunk 1 above)
Comment (bhushan-nemade-dt) [reply docs/user-guide/plugins/container-image-exporter.md]: Fixed.

**Review by abhibhaw — APPROVED** [review]
LGTM

**Review by ashoknayak777 — APPROVED** [review]

**Pull request merged**